src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/PublicationViewModels.cs

```diff
@@ -11,42 +11,49 @@ public class PublicationViewModel
     {
         public Guid Id { get; set; }
 
-        public string Title { get; set; }
+        public string Title { get; set; } = Empty;
 
-        public string Summary { get; set; }
+        public string Summary { get; set; } = Empty;
 
-        public string Slug { get; set; }
+        public string Slug { get; set; } = Empty;
 
-        public List<ReleaseViewModel> Releases { get; set; }
+        public IdTitleViewModel Topic { get; set; } = null!;
 
-        public List<TitleAndIdViewModel> Methodologies { get; set; }
+        public IdTitleViewModel Theme { get; set; } = null!;
 
-        public ExternalMethodology ExternalMethodology { get; set; }
-
-        public Guid TopicId { get; set; }
-
-        public Guid ThemeId { get; set; }
-
-        public ContactViewModel Contact { get; set; }
+        public ContactViewModel Contact { get; set; } = null!;
```

Comment:
I think you're right but I'll defer to Nick on that. If it is, my preference would be to do that as a separate PR, much like the new external methodology endpoints.

Reply:
Yep happy for you to split this out. Originally I didn't spec out tickets for contact or external methodologies because it didn't feel like it was a necessity to getting all the admin dashboard over the line.

Now that we have the `Publication.Summary` field that's required, it feels like we at least need endpoints to update these entities as we'd otherwise run into validation errors.

Reply:
Created ticket EES-3695